jupyterlab_bigquery/jupyterlab_bigquery/handlers.py

```diff
@@ -179,3 +187,24 @@ def post(self, *args, **kwargs):
               'message': str(e)
           }
       })
+
```

Comment:
Tests for this and other handlers will be coming up once @hwing777 is done reorganizing the handlers files